Classes/Emogrifier.php

```diff
@@ -216,7 +222,10 @@ function (array $m) {
         // (these blocks should be appended so as to have precedence over conflicting styles in the existing CSS)
         $allCss = $this->css;
 
-        $allCss .= $this->getCssFromAllStyleNodes($xpath);
+        // Only parse the <style> blocks if that option is enabled
```

Comment:
This comment is not necessary anymore either as the code already is clear enough.